kratos/elements/distance_calculation_element_simplex.h

```diff
@@ -33,6 +33,7 @@
 
 // Application includes
 #include "includes/variables.h"
+#include "includes/kratos_flags.h"
```

Comment:
Why the additional incudes?

Reply:
you get a compilation error without it as it uses BOUNDARY flag (kratos_flags is not included in kratos_components either)

Reply:
Ok last question, is it needed in the header or could it be in the cpp?

Reply:
there is no cpp for this two elements they are both header only